heron/stmgr/src/cpp/manager/instance-server.cpp

```diff
@@ -463,6 +483,13 @@ void InstanceServer::StopBackPressureConnectionCb(Connection* _connection) {
   sp_string instance_name = GetInstanceName(_connection);
   CHECK_NE(instance_name, "");
 
+  LOG(INFO) << "We don't observe back pressure now on sending data to instance " << instance_name;
```

Comment:
I am just wondering if this should be a WARNING log. Not a blocker.

Reply:
Sounds good